### DISCUSSION TITLE
Support for UDP protocol in v2.18.0

### DISCUSSION THREAD

**Discussion by Samuel-John** [COVESA/dlt-viewer#563] [Q&A] [answered]
Hi team,
I am using v2.18.0. I wanted to know if this version of dlt-viewer supports UDP protocol.
While adding ECU, the interface type has the option to select UDP but unlike the latest version there is no separate tab for UDP. Instead there is IP.
I have provided the correct IP in Hostname and the correct IP Port.
Yet, the DLT message is not visible in the dlt-viewer.

Can somebody please let me know what is causing this?

Thanks in advance.

**Comment by alexmucde** [comment, marked as answer]
@Samuel-John v2.18.0 is very old and we had some issues with UDP reception in these old version, vene with performance. Please switch to latest stable one 2.26.0

**Reply by Samuel-John** [reply]
Thanks @alexmucde .

_Just an info: Since you stopped using 'v' in release tags, the latest versions are not seen on top of the Tags in Github._ ;-)
